lib/modules/manager/gitlabci/extract.ts

```diff
@@ -115,11 +207,26 @@ export function extractPackageFile(
       }
       deps = deps.filter(is.truthy);
     }
+
+    const includedComponents = getAllIncludeComponents(doc);
+    for (const includedComponent of includedComponents) {
+      const dep = extractDepFromIncludeComponent(includedComponent, endpoint);
+      if (dep) {
+        deps.push(dep);
+      }
+    }
   } catch (err) /* istanbul ignore next */ {
-    logger.debug(
-      { err, packageFile },
-      'Error extracting GitLab CI dependencies',
-    );
+    if (err.stack?.startsWith('YAMLException:')) {
```

Comment:
🤔 I don't really see a benefit for this conditional logging. only message is different 

@rarkins WDYT?

Reply:
Not too ambitious about it. I have copied it from the existing code in the `gitlabci-include` manager.

Reply:
This pattern is used in multiple places:

<img width="377" alt="image" src="https://github.com/renovatebot/renovate/assets/1540469/231cf130-7ddc-44ce-a61c-0173e928bb09">

Reply:
But agreed, the different message does not bring much benefit, unless someone explicitly wants to match for it in their logging.